doc/release-notes/4259-java-11.md

```diff
@@ -6,34 +6,41 @@ The Dataverse software now runs and requires Java 11. This will provide performa
 
 ## Notes to Admins
 
-### Java 11
+### Java 11 Upgrade
 
 There are some things to note and keep in mind regarding the move to Java 11:
 
-- You should install the JDK/JRE following your usual methods, depending on your operating system.
+- You should install the JDK/JRE following your usual methods, depending on your operating system. An example of this is:
+
+    `$ sudo yum remove java-1.8.0-openjdk java-1.8.0-openjdk-devel java-1.8.0-openjdk-headless`  
+
+    `$ sudo yum install java-11-openjdk-devel`  
+    
+    The `remove` command may provide an error message if -headless isn't installed.
+
 - We targeted and tested Java 11, but 11+ will likely work. Java 11 was targeted because of its long term support.
-- If you're moving from a Dataverse installation that was previously running Dataverse 4.x, wou will need to adjust some JVM options in domain.xml as part of the upgrade process. We've provided these optional steps below. These steps are not required if you first installed Dataverse version was in the 5.x series.
+- If you're moving from a Dataverse installation that was previously running Dataverse 4.x, you will need to adjust some JVM options in domain.xml as part of the upgrade process. We've provided these optional steps below. These steps are not required if your first installed Dataverse version was in the 5.x series.
```

Comment:
It might be more accurate to specify this relates to running on GF4 vs P5? In practice it's almost definitely the same, but I guess it could be possible someone installed Dataverse 5 on GF4 first (whether it worked or not) and then upgraded the server via upgrade instructions.

Reply:
@scolapasta we could try to change this, not sure if it's more confusing though, and I understand this is a possible but not supported case. Here's what I came up with in case we do want to go in this direction:

"If you're moving from a Dataverse installation that was previously running Glassfish 4.x (typically this would be Dataverse 4.x), you will need to adjust some JVM options in domain.xml as part of the upgrade process. We've provided these optional steps below. These steps are not required if your first installed Dataverse version was running Payara 5.x (typically Dataverse 5.x)."

Reply:
While it may be overkill, I like this version better.

Reply:
addressed in https://github.com/IQSS/dataverse/pull/7544/commits/698ef34d56f8d8aacbba8f2111ebf80639aeb22c